static/css/restyle/restyle.less

```diff
@@ -566,6 +566,10 @@ body:not(.home) .amo-header,
   margin-top: 22px;
   padding: 15px 45px 15px 15px;
 
+  .html-rtl & {
```

Comment:
There is a small typo/mistake. It should be `.html-rtl&` instead of `.html-rtl &`